mappings/net/minecraft/world/gen/OreVeinCreator.mapping

```diff
@@ -0,0 +1,27 @@
+CLASS net/minecraft/unmapped/C_kryzafwp net/minecraft/world/gen/OreVeinCreator
+	FIELD f_ftqfbidn VEIN_THRESHOLD F
+	FIELD f_iwbpmpmn MIN_RICHNESS F
+	FIELD f_oagwymmv MAX_RICHNESS F
+	FIELD f_pnrsifvr SKIP_ORE_WHEN_GAP_NOISE_BELOW F
```

Comment:
```suggestion
	FIELD f_pnrsifvr MIN_GAP_NOISE_ORE_SKIP_THRESHOLD F
```
This is still verbose (I don't understand the field enough to suggest something less verbose), but the format is more similar to the other fields.